recipes/adapter-transformers/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "adapter-transformers" %}
+{% set version = "3.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/adapter-transformers-{{ version }}.tar.gz
+  sha256: 0004a417b3ebb7a40c3bf666a0ad6a69bdacc384345d68288edb6c7213aaabd7
+
+build:
+  skip: true  # [py<38]
```

Comment:
I think we can drop the `skip` and make this `noarch: python`. See:
https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

You'll then change `python` to `python >= 3.8` under the `host` and `run` requirements. The also drop the version restriction on `importlib-metadata` below.